clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/securitygroups/SecurityGroupRule.groovy

```diff
@@ -17,9 +17,7 @@
 package com.netflix.spinnaker.clouddriver.model.securitygroups
 
 import com.netflix.spinnaker.clouddriver.model.SecurityGroup
-import groovy.transform.Immutable
 
-@Immutable
```

Comment:
Dropped because this wasn't being enforced. It was falsely assuming that `SecurityGroup` was immutable which wasn't made apparent to the runtime until `SecurityGroup` was converted to Java.